blueprints/data-solutions/data-platform-foundations/demo/dataflow-csv2bq/README.md

```diff
@@ -0,0 +1,58 @@
+## Pipeline summary
+This demo serves as a simple example of building and launching a Flex Template Dataflow pipeline. The code mainly focuses on reading a CSV file as input along with a JSON schema file as side input. The pipeline Parses both inputs and writes the data to the relevant BigQuery table while applying the schema passed from input.
+
+![Dataflow pipeline overview](../../images/df_demo_pipeline.png "Dataflow pipeline overview")
+
+## Example build run
+
+Below is an example for triggering the Dataflow flex template build pipeline defined in `cloudbuild.yaml`. The Terraform output provides an example as well filled with the parameters values based on the generated resources in the data platform.
+
+```
+GCP_PROJECT="ff-orc"
+TEMPLATE_IMAGE="[REGION].pkg.dev/[ORCHESTRATION-PROJECT]/[REPOSITORY]/csv2bq:latest"
+TEMPLATE_PATH="gs://[DATAFLOW-TEMPLATE-BUCKEt]/csv2bq.json"
+STAGIN_PATH="gs://[ORCHESTRATION-STAGING-BUCKET]/build"
+LOG_PATH="gs://[ORCHESTRATION-LOGS-BUCKET]/logs"
+REGION="[YOUR-REGION]"
+
+gcloud builds submit \
```

Comment:
Once created a dedicated SA, I would impersonate that SA.

Reply:
Done.

Reply:
This gcloud command is still not using the SA. I would explicitly use the --impersonate-service-account flag for the Google Cloud CLI.